doc/generate_doc.sh

```diff
@@ -1,6 +1,6 @@
 # build doc
-export WORK_ROOT=$(cd `dirname "$0"`;pwd)
-export BIGFLOW_PYTHON_HOME=$WORK_ROOT/../bigflow_python/python/
+export WORK_ROOT=-$(cd `dirname "$0"`;pwd)
+export BIGFLOW_PYTHON_HOME=${BIGFLOW_PYTHON_HOME:-$WORK_ROOT/../bigflow_python/python/}
```

Comment:
"$WORK_ROOT/../bigflow_python/python/" no matter under what circumstances should be added? In case there is no such value in BIGFLOW_PYTHON_HOME?

Reply:
Building doc will set BIGFLOW_PYTHON_HOME，building will not set BIGFLOW_PYTHON_HOME and "$WORK_ROOT/../bigflow_python/python/" will be used.